### PR TITLE
Completed javascript-data-model exercise. -Brandon Moy

### PR DESCRIPTION
**Pull request by brandon-moy**
image

### CODE REVIEW

File: javascript-data-model/order-history.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      shipTo: 'JS Masher',
+      orderNum: '113-2883177-2648248',
+      title: 'The Art of Sql',
+      author: 'Faroult, Stephane',
+      deliveryDate: 'Jul 5, 2020',
+      returnDate: 'Aug 4, 2020',
+      itemPrice: 33.99,
+      img: ''
+    }
+  ]
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js:58]: All of your other orders are a single object, but this order is an array. That is not a consistent format, and will lead to bugs.

File: javascript-data-model/order-history.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    orderPlaced: 'August 4, 2020',
+    shipTo: 'JS Masher',
+    orderNum: '114-3941689-8772232',
+    title: 'JavaScript for impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    deliveryDate: 'Aug 8, 2020',
+    returnDate: 'Sep 7, 2020',
+    itemPrice: 31.55,
+    img: ''
+  },
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js:12]: With this setup, you are intermixing data about the order and data about the item(s) in the order. I would try to find a way to store data about the order, and have one piece of that data be the information about the item(s) ordered.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-data-model/order-history.js (hunk 1 above)
Comment (brandon-moy) [reply javascript-data-model/order-history.js:58]: Thanks, Cody! Removed the nested array for consistent format.
image

File: javascript-data-model/order-history.js (hunk 2 above)
Comment (brandon-moy) [reply javascript-data-model/order-history.js:12]: Thanks Cody, tried a couple of things and made the main object house the order data and then one of the properties of that object be another object to store the information on the item.
image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**